serverless/admin_guide/serverless-admin-metrics.adoc

```diff
@@ -0,0 +1,40 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-admin-metrics"]
+= Metrics
+include::modules/common-attributes.adoc[]
+:context: serverless-admin-metrics
+
+toc::[]
+
+Metrics enable cluster administrators to monitor how {ServerlessProductName} cluster components and workloads are performing.
+
+[id="prerequisites_serverless-admin-metrics"]
+== Prerequisites
+
+* See the {product-title} documentation on xref:../../monitoring/managing-metrics.adoc#managing-metrics[Managing metrics] for information about enabling metrics for your cluster.
+* To view metrics for Knative components on {product-title}, you need cluster administrator permissions, and access to the web console *Administrator* perspective.
+
+// Common metrics
```

Comment:
Should the Common Metrics be commented out since there are include statements following it?

Reply:
Yeah, this is just a reference for us to know that the following modules are common to both Knative components, it's not needed to show up in the doc